main.go

```diff
@@ -25,6 +31,8 @@ type RKE2Config struct {
 	TLSSan []string `yaml:"tls-san"`
 }
 
+var configScanDir = []string{"/oem", "/usr/local/cloud-config", "/run/initramfs/live"}
```

Comment:
cluster provider plugins should not read configuration directly the configuration will be passed by kairos.

Reply:
the cluster object in the provider does not contain the env config values.